Modules/Project/Services/ProjectService.php

```diff
@@ -283,6 +287,38 @@ private function updateProjectTechstack($data, $project)
         }
     }
 
+    private function updateProjectRequirement($data, $project)
+    {
+        if (isset($data['designation'])) {
+            $needed = $data['needed'];
+
+            $designationMap = config('project.designation');
+            $project_id = $project->id;
```

Comment:
Please use correct naming convention
```suggestion
            $projectId = $project->id;
```